ci/scripts/start_instance.sh

```diff
@@ -78,9 +84,9 @@ while true; do
       --zone=${ZONE} \
       --machine-type=custom-${CPUS}-${RAM_MEGABYTES} \
       --min-cpu-platform=Intel\ Skylake \
-      --network="heavy-lifters" \
-      --subnet="heavy-lifters" \
-      --image-family="${IMAGE_FAMILY_PREFIX}geode-builder" \
+      --network="heavy-lifters-2" \
```

Comment:
Would prefer 'heavy-lifters-win' as an identifier. Easier to understand when browsing through the GCP console

Reply:
This would apply to all heavy lifters - linux as well as windows. This moves them into the non-overlapping `10.1.0.0` network.